### ISSUE TITLE
Feature Request: DNS Blacklist

### ISSUE THREAD

**Issue by mastad0n** [getdnsapi/stubby#65]
WIth DNSCrypt it looked like this:

https://web.archive.org/web/20180108142429/https://webcache.googleusercontent.com/search?q=cache:https://github.com/jedisct1/dnscrypt-proxy/wiki/Blacklists

Is this possible with Stubby?

**Comment by saradickinson** [comment]
No, sorry - not possible today with only stubby. But thanks for the feature request, we'll look into it.

**Comment by ibksturm** [comment]
any news about blacklisting?

**Comment by saradickinson** [comment]
We haven't had any time to work on this but have a planning meeting next week so we'll talk about it then. But the upvote is noted :-)

**Comment by ibksturm** [comment]
Hi @saradickinson and, some news?

**Comment by saradickinson** [comment]
Hi @ibksturm we are hoping to get this into a future release soon as we realise it is useful. We're working on a major re-factor of the connection handling for 1.5 which is taking most of our effort at the moment but we will try to add this in a 1.4 release if we get chance.

**Comment by SantjagoCorkez** [comment]
Any updates on this?

**Comment by saradickinson** [comment]
@SantjagoCorkez It is still on the TODO list, but other work has taken higher priority for what resources we have. Thanks for the bump though to remind us this is still needed!

**Comment by YellowRoseCx** [comment]
Would be great to have this feature
